test/Microsoft.AspNet.Mvc.ModelBinding.Test/Metadata/DataMemberBindingValidationMetadataProviderTest.cs

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Runtime.Serialization;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding.Metadata
+{
+    public class DataMemberBindingMetadataProviderTest
```

Comment:
also, shouldn't this class be named `DataMemberRequiredBindingMetadataProviderTest` ?